server/planning/events/events_post.py

```diff
@@ -208,20 +208,20 @@ async def post_event(self, event, new_post_state, repost):
         plannings = get_related_planning_for_events([event[ID_FIELD]], "primary")
 
         event["plans"] = [p.get("_id") for p in plannings]
-        self.publish_event(event, version)
+        await self.publish_event(event, version)
 
         if len(plannings) > 0:
             failed_planning_ids = await self.post_related_plannings(plannings, new_post_state)
 
         return updated_event.to_dict(), failed_planning_ids
 
-    def publish_event(self, event, version):
+    async def publish_event(self, event, version):
         # check and remove private contacts while posting event, only public contact will be visible
         event["event_contact_info"] = [try_cast_object_id(contact["_id"]) for contact in get_contacts_from_item(event)]
 
         """Enqueue the items for publish"""
-        # TODO-ASNYC - Change to async when `published_planning` is converted to async
-        version_id = get_resource_service("published_planning").post(
+        # TODO-ASYNC - Change to async when `published_planning` is converted to async
```

Comment:
Is this comment needed anymore?

Reply:
I left it because I thought we're still going to migrate these to full async services (with pydantic and all the toys) in the future but we can remove it if that's not the case.